extras/rya.streams/api/src/main/java/org/apache/rya/streams/api/queries/InMemoryQueryRepository.java

```diff
@@ -269,6 +276,7 @@ private void updateCache() {
 
     @Override
     protected void runOneIteration() throws Exception {
+        checkState();
```

Comment:
This check isn't needed since the service will never invoke this method without being started.

Reply:
right.  just saw the updateCache() call and added the check as a reflex